uint8 MISLOCALIZED=0
uint8 MOST_PROBABLY_LOCALIZED=1
uint8 PERFECTLY_LOCALIZED=2

time stamp
uint8 status
string text
float32 det_cov
